js/Domupdates.js

```diff
@@ -189,16 +191,28 @@ let domUpdates = {
   handleConsonantGuessed(e) {
     e.preventDefault();
     let guessLetterInput = this.consonantLetterInput.value.toLowerCase().trim();
+    let correctGuess = game.correctGuess(guessLetterInput.toUpperCase());
     if (this.vowels.includes(guessLetterInput)) {
       // TODO: change UI to some message instead of alert
       alert('PLEASE ENTER A CONSONANT!!!');
     } else {
+      this.revealLetters(correctGuess, guessLetterInput);
       this.greyOut(guessLetterInput);
       game.handleConsonantGuessed();
       this.showPlayerScore();
     }
   },
 
+  revealLetters(correctGuess, letter) {
+    if (correctGuess) {
+      [...this.greenPuzzleBoxes].filter((currentBox) => {
```

Comment:
This looks great, but I'm a little confused by this spread operator. Let's chat in person about it today.